portfolio/src/main/webapp/style.css

```diff
@@ -1,65 +1,263 @@
-#menu {
-    color: white;
+#home-page {
+    font-family: "Lato", sans-serif;
+    /*background-image: url("/images/bg.jpg");*/
+}
+
+.home-image {
+    bottom: 0px;
+    height: 100%;
+    left: 800px;
+    position: absolute;
+    top: 0px;
+    width: 700px;
+}
+
+.intro {
+    font-size: 170%;
+    left: 700px;
     position: relative;
-    text-align: center;
+    text-align: justify;
+    top: 160px;
+    width: 500px;
 }
 
-#menuimage {
-    height: 800px;
-    width: 100%;
+.top-cloud {
+    height: 320px;
+    position: absolute;
+    top: 30px;
+    width: 650px;    
 }
 
-#name {
-    color: white;
-    font-size: 600%;
-    left: 60px;
+#bottom-cloud {
+    height: 320px;
     position: absolute;
-    text-shadow: 2px 2px 5px black;
+    top: 360px;
+    width: 650px;
+}
+
+#sub-page {
+    font-family: "Lato", sans-serif;
```

Comment:
That's a good practice what you did here, to have a fallback font chosen in case the first one is not available.